### DISCUSSION TITLE
Migrate to official forum?

### DISCUSSION THREAD

**Discussion by samdark** [yiisoft/yii2#19888] [General]
GitHub discussions are alright but we have a nice official forum at https://forum.yiiframework.com/ that supports GitHub auth so instead of further splitting community it could be a good idea to close discssions here and, instead, move to the forum.

What do you think?

**Comment by bizley** [comment]
Let's. This should put possibly a new life into our forum.

**Comment by simialbi** [comment]
I prefer GitHub, a browser tab less 🤔, an account less, notifications on updates etc.

**Comment by rhertogh** [comment]
@samdark Does the forum has options like direct messaging and private forums? That would be an advantage.

**Reply by samdark** [reply]
Yes.

**Comment by tomaszkane** [comment]
Does GH Discussions has any "game changer" feature that is not present in forum?
If not, then forum sounds like a better place. For me at this moment both is fine.

**Reply by samdark** [reply]
Nope, no "game changer" features.

**Comment by forwzb** [comment]
建议回到论坛，相比github更愿意在自己的论坛去参与交流，可以给官网增加活跃度，如果后期发现有更好的交流方式，论坛也好修改，但是在github上不行。
